src/components/homepage/Notifcations.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { Alert } from '@mantine/core';
+import { AlertCircle } from 'tabler-icons-react';
+
+// Modify these fields to change the outage message
+const statusmsg = [
+    {
+        title: "Cluster Outage: This is for users of the MOC clusters!",
+        description: "As a reminder these clusters (Smaug, Infra, Curator) will be unavailable from Monday, May 23, 2022 at 6PM through Thursday, May 26, 2022 at 12PM. Due to a scheduled power outage.",
```

Comment:
Ideally we wouldn't need to update the alert in code and rebuild the website each time announcements are updated.

But at the minimum I think it should be exported out to it's own file. 

Ideally: It would query some external source for this information. Maybe run an async query for issues in the `operate-first/operate-first.github.io` repo with label `label:announcement` and use their titles as the announcement if it finds any?

@larsks @durandom wdyt?

Reply:
Actually the gatsby doc has a section on this and uses GH as an example :)

https://www.gatsbyjs.com/docs/conceptual/data-fetching/#retrieving-data-with-the-fetch-api